content/en/blog/2022/k8s-otel-expose/index.md

```diff
@@ -218,38 +218,38 @@ metadata:
 spec:
   acme:
     server: https://acme-v02.api.letsencrypt.org/directory
-    email: <REPLACE: your e-mail address, e.g.: "someone@example.com">
+    email: your-email-address-here@example.com # REPLACE
```

Comment:
The `<REPLACE: your ...>` (used here and two places below), confused even Prettier. This way of illustrating an entry that has parts to be replaced seems better / less confusing for the pretty-printers.